docs/using.rst

```diff
@@ -272,22 +272,22 @@ you prefer to manage everything by hand, this section provides
 information on where to find necessary files.
 
 All generated keys and issued certificates can be found in
-``/etc/letsencrypt/live/$domain``. Rather than copying, please point
+``/etc/certbot/live/$domain``. Rather than copying, please point
 your (web) server configuration directly to those files (or create
-symlinks). During the renewal_, ``/etc/letsencrypt/live`` is updated
+symlinks). During the renewal_, ``/etc/certbot/live`` is updated
 with the latest necessary files.
 
-.. note:: ``/etc/letsencrypt/archive`` and ``/etc/letsencrypt/keys``
+.. note:: ``/etc/certbot/archive`` and ``/etc/certbot/keys``
    contain all previous keys and certificates, while
-   ``/etc/letsencrypt/live`` symlinks to the latest versions.
+   ``/etc/certbot/live`` symlinks to the latest versions.
```

Comment:
All `/etc/certbot` paths above are incorrect. I believe we decided not to make this change now (or maybe ever). See #2830.